make_mock_json.py

```diff
@@ -0,0 +1,401 @@
+            'release': TEST_BUILD,
+            'platform': "x86_64",
+            'arrangement_version': DEFAULT_ARRANGEMENT_VERSION,
+            'scratch': True,
+        }
+
+        build_name = self.create_a_mock_build(self.osbs.create_worker_build, TEST_BUILD, "",
+                                              build_kwargs)
+        build_path = "/".join([self.mock_dir, build_name])
+        if os.stat(build_path):
+            created_path = "/".join([self.mock_dir,
+                                    "created_build_config_test-build-config-123.json"])
+            # build_test-build-config-123.json and created_build_config... are identical
+            # and os doesn't have a cp command, so create a link
+            os.link(build_path, created_path)
+
+        del build_kwargs['platform']
+
+        self.create_a_mock_build(self.osbs.create_orchestrator_build, TEST_ORCHESTRATOR_BUILD,
+                                 "orchestrator-", build_kwargs)
+
+    def create_mock_imagestream(self):
+        print("look in {}".format(self.capture_dir))
+        #imagestream_repo =  "quay.io/prometheus/prometheus"
+        imagestream_server = "registry-proxy-stage.engineering.redhat.com"
+        imagestream_file = "/prometheus/prometheus"
+        imagestream_repo = imagestream_server + imagestream_file
+        try:
+            imagestream = self.osbs.create_image_stream(TEST_IMAGESTREAM, imagestream_repo)
+        except:
+            print("imagstream {} already exists.".format(TEST_IMAGESTREAM))
+            print("run `oc delete imagestream {}` and try again".format(TEST_IMAGESTREAM))
+            exit(0)
+        tags = ['latest', 'master', 'v2.13.0']
+        #tags = ['latest', 'master', '7.2.username-66', '7.2.username-67']
+        self.osbs.import_image_tags(TEST_IMAGESTREAM, tags, imagestream_repo)
+        for tag in tags:
+            self.osbs.ensure_image_stream_tag(imagestream.json(), tag)
+
+        imagestreamimport_fname = "post-namespaces_osbs-stage_imagestreamimports_-000.json"
+        imagestreamimport_path = "/".join([self.capture_dir, imagestreamimport_fname])
+        imagestreamimport_data = []
+        imagestream_data = []
+        with open(imagestreamimport_path, "r") as infile:
+            imagestreamimport_data = json.load(infile)
+
+        imagestreamimport_data["spec"] = {
+            "images": [
+                {
+                    "to": {
+                        "name": "latest"
+                    },
+                    "referencePolicy": {
+                        "type": "Source"
+                    },
+                    "from": {
+                        "kind": "DockerImage",
+                        "name": "example.com/lucarval/rsyslog:latest"
+                    },
+                    "importPolicy": {
+                        "insecure": True
+                    }
+                }
+            ],
+            "import": True
+        }
```

Comment:
Is this just to de-identify it? Is it worth it, if we're using a publicly-available repository?

Reply:
This was to make stuff work with existing tests, but it turns out to be unnecessary.  Removed.